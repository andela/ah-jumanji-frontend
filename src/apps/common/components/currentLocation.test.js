import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import ShowTheLocation from'./currentLocation';


Enzyme.configure({ adapter: new Adapter() });

describe('ShowTheLocation', () => {
  it('should render correctly', () => {
    const component = shallow(<ShowTheLocation location="home" />);

    expect(component).toMatchSnapshot();
  });
});
