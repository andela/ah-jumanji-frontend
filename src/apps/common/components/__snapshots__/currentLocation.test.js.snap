// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowTheLocation should render correctly 1`] = `
<Route>
  <Component />
</Route>
`;
